zone/zone.cpp

```diff
@@ -330,81 +330,154 @@ bool Zone::LoadGroundSpawns() {
 	return(true);
 }
 
+void Zone::DumpIt(uint32 npcid) {
```

Comment:
This need to be more specific to merchants, eg: `Zone::DumpMerchantList`

Reply:
Good catch.  That was going to be a temporary function, but I figured leave it in, in case details are needed in the future.  Fixing.

Reply:
Could someone also request a review from @Uleat ?  I don't have the ability to do that formally.